guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md

```diff
@@ -3,7 +3,18 @@ title: Connect Redux to React
 ---
 ## Connect Redux to React
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Solution
+<details>
+  <summary>Spoiler!</summary>
+  
+```jsx
+// change code below this line
+const ConnectedComponent = connect(mapStateToProps, mapDispatchToProps) (Presentational)
+```
+
+</details>
+
+<a href='https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
 
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
```

Comment:
```suggestion
```
Remove this line and all lines like this from all the 9 files you're making changes to

Reply:
I just removed the links to github repo. Are they as desired now?
By the way, someone had provided solutions to the remaining challenge, so I just amended 9 files~
I am not familiar with how to "pull the branch locally and squash commits", hope it is still fine?

Reply:
It is fine, although, if you Google how to squash commits, it'll be easier

Reply:
LGTM